### ISSUE TITLE
testing wheels fails in some cases

### ISSUE THREAD

**Issue by fschulze** [devpi/devpi#585]
When testing wheels for packages where the location of the package is the same as the package name and thus importable the wheel isn't properly installed and the tests fail.

For projects which have their packages in a ``src`` sub directory this isn't an issue.

From the tox logs it looks like this is a change in pip, as it says the dependency is already satisfied by the ``packagename`` folder in ``.../temporarytestfolder/targz/packagename`` when it should install the downloaded wheel. For packages with ``src`` it installs it normally.

**Comment by hpk42** [comment]
so how to bite into the apple and move devpi-{server,client,...} one level down to src?

**Comment by fschulze** [comment]
That would solve it for us, but this may affect more people.

**Comment by hpk42** [comment]
do you have any idea for how to solve this? it just seems that tox could bail out if it can detect it.

**Comment by jrast** [comment]
@fschulze what is the behaviour of this bug? Is the package not importable and the tests fail with something like "ModuleNotFoundError: ..."?

Right now i needed to stop building wheels because all my devpi tests failed with the "ModuleNotFoundError".

**Comment by fschulze** [comment]
@jrast yep, exactly that. But this needs to be fixed in either pip (unlikely I guess, though it's a regression, but I haven't investigated why the change was made) or in tox by using another working directory.

**Comment by jrast** [comment]
Thank you for the clarification. Is there a issue for tox which I can monitor?

**Comment by fschulze** [comment]
I don't think I added one, otherwise I would most likely have linked to here. I haven't checked the tox issue tracker yet.
